app/views/templates/root_node.html.haml

```diff
@@ -29,12 +29,14 @@
 
 
   %h2.home-heading News
-
+  - if NewsArticle.published.all.empty?
```

Comment:
Feels wrong to have ActiveRecord stuff in a template. Maybe have `news_articles` getter in the RootNode class instead? Same for `ministers_count`